pkg/controller/operators/openshift/helpers_test.go

```diff
@@ -525,7 +464,7 @@ func TestIncompatibleOperators(t *testing.T) {
 			scheme := runtime.NewScheme()
 			require.NoError(t, AddToScheme(scheme))
 
-			fcli := fake.NewClientBuilder().WithScheme(scheme).WithObjects(objs...).Build()
+			fcli := fake.NewClientBuilder().WithObjects(objs...).WithScheme(scheme).Build()
```

Comment:
Why did we swap these two calls?

Reply:
I must have done this on accident while working. I'll revert it to minimize the diff.